subcommittees/mentoring/procedure-debriefing-session.md

```diff
@@ -0,0 +1,41 @@
+# Procedure - Debriefing Session
+
+This document intends to describe how debriefing sessions
+are organized so that it continue even after members go away.
+
+## Terminology
+
+-   **Debriefing session**: is an online meeting
+    where instructors could share experience about the workshops
+    they taught.
+
+## We Need a Debriefing Session
+
+The Mentoring Subcommittee is responsible to organize
+the debriefing sessions so that
+
+-   each month has at least one debriefing session and
+
+-   instructors are notify with at least one week in advance.
+
+## Inviting Instructors
+
+The invitation should be send directly to instructors by email.
+The contact address of the instructors are available at
+https://amy.software-carpentry.org/workshops/debrief.
+
+Optionally,
+the invitation could be send to
+[Instructors mailing list](http://lists.software-carpentry.org/mailman/listinfo/instructors_lists.software-carpentry.org).
+
+## Show time
+
+Each debriefing session **must** be hosted
+by at least one member of the Mentoring Subcommittee.
+
+## Report to the Community
+
+After each debriefing session,
+one member of the Mentoring Subcommittee
+that attended the debriefing session
+**must** wrote a blog post summarizing the topics discussed.
```

Comment:
How about adding a list of the questions we usually have listed on the etherpad? That way we have a (permanent) record of the things we ask.